application/templates/twoColumnPage.njk

```diff
@@ -36,11 +55,11 @@
       <div class="app-subnav">
 
         <ul class="app-subnav__section">
```

Comment:
You could probably add an if statement around this `ul` as well. It shouldn't ever be the case that there are no items but might help if we restructure things in the future.